access/acceptance/secret_acl_test.go

```diff
@@ -26,6 +27,9 @@ func TestAccSecretAclResource(t *testing.T) {
 	scope := fmt.Sprintf("tf-scope-%s", acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum))
 	principal := "users"
 	permission := "READ"
+	client := common.CommonEnvironmentClient()
+	me, _ := identity.NewUsersAPI(client).Me()
```

Comment:
this is called "ignoring the error". we must handle errors everywhere, assert.NoError(t,err) in tests as well.

Reply:
just pushed the modified version, but it was already merged - will fix in the next iteration, when will do #235